src/lib/components/Picker.react.js

```diff
@@ -0,0 +1,105 @@
+// @flow
+
+import React, {Component} from 'react';
+import DateFnsUtils from '@date-io/date-fns';
+import {
+  MuiPickersUtilsProvider,
+  KeyboardTimePicker,
+  KeyboardDatePicker,
+} from '@material-ui/pickers';
+
+type Props = {
+  /** Picker ID */
+  id: string,
+  /** Label for the date or time picker */
+  label?: string,
+  /** Type of date or time picker, "time", "date", or "date-dialog" */
+  type?: string,
+  /** Representation of datetime, like 2020-12-25T13:11:00.000Z */
+  value?: string,
+  /** Dash callback to update props on the server */
+  setProps?: () => void,
+};
+
+type State = {
+  open: boolean,
+};
+
+const defaultProps = {
+  label: "",
+  value: "01-01-2020",
+  type: "date",
+  setProps: () => {},
+};
+
+export default class Picker extends Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+    this.state = {open: props.open};
+  }
+
+  UNSAFE_componentWillReceiveProps(nextProps: Props, nextContext: *): void {
+    if (nextProps.value !== this.state.value)
+      this.setState({value: nextProps.value})
+  }
+
+  handleChange = (event) => {
+    const selectedDate = event
+    this.setState({value: selectedDate });
+
+    if (typeof this.props.setProps === 'function') {
+      this.props.setProps({value: selectedDate});
+    }
+  };
+
+  render() {
+    const {id, label, type, value} = this.props;
+    const selectedDate = new Date(value);
+
+    if(type === "date"){
+      return (
+      <MuiPickersUtilsProvider utils={DateFnsUtils}>
+          <KeyboardDatePicker
+            disableToolbar
+            variant="inline"
+            format="MM/dd/yyyy"
```

Comment:
This does not match the YYYY-MM-DD format shown in the comment for the `value` prop. Can we change this and the following `format` to match it?

Reply:
This `format` is the output, unfortunately it will not allow us to match this for this input. Should we allow users to choose the output format?

Reply:
So the input format cannot change? In that case, I think we should make everything match that.

Reply:
Correct. That would mean this would read `2020-12-25`, is that what we want?
![image](https://user-images.githubusercontent.com/47650493/102922163-92c48f00-445b-11eb-88e3-b2ba679760ee.png)

Reply:
I prefer that, but perhaps flexibility here is better. Would you implement your idea to let users choose the format? That will make it easier down the road to change, or even make a setting to control this.

Reply:
Okay. I will allow users to change it, but default to the input format